common/initial_commit_tx.h

```diff
@@ -28,28 +28,32 @@ static inline size_t commit_tx_base_weight(size_t num_untrimmed_htlcs,
 					   bool option_anchor_outputs)
 {
 	size_t weight;
+    size_t num_outputs;
 
 	/* BOLT #3:
 	 *
 	 * The base fee for a commitment transaction:
 	 *  - MUST be calculated to match:
 	 *    1. Start with `weight` = 724 (1124 if `option_anchors` applies).
 	 */
-	if (option_anchor_outputs)
+	if (option_anchor_outputs) {
 		weight = 1124;
-	else
+        num_outputs = 4;
+	} else {
 		weight = 724;
-
+        num_outputs = 2;
+    }
```

Comment:
looks like a couple of tabs are missing?

Reply:
fixed, thanks vim